framework/src/org/checkerframework/framework/util/defaults/QualifierDefaults.java

```diff
@@ -199,6 +199,19 @@ public String toString() {
     }
 
     /**
+     * Check that we have defaults specified.
+     *
+     * @return whether we found a Default with location OTHERWISE or ALL
+     */
+    public boolean hasDefaultsForCheckedCode() {
+        for (Default def : checkedCodeDefaults) {
+            if (def.location == TypeUseLocation.OTHERWISE || def.location == TypeUseLocation.ALL)
```

Comment:
I would also put `{}` around the `return true;`, to be more uniform and ease future changes.